sunpy/net/_attrs.py

```diff
@@ -78,7 +80,7 @@ def pad(self, timedelta):
         return type(self)(self.start - timedelta, self.start + timedelta)
 
     def __repr__(self):
-        return '<Time({s.start!r}, {s.end!r}, {s.near!r})>'.format(s=self)
+        return f'<sunpy.net.attrs.Time({self.start.iso}, {self.end.iso}, {self.near.iso if self.near else ""})>'
```

Comment:
You dont' have to add this because you have to change all the doctests but this:
```suggestion
        return f'<sunpy.net.attrs.Time({self.start.iso}, {self.end.iso}{", " + self.near.iso if self.near else ""})>'
```
would be nicer.